libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -758,6 +791,17 @@ where
         }
     }
 
+    /// Adds outbound connection to matrix life cycle after it has been authorized
+    /// these connections are cannot be reconnected when dropped or lost.
+    ///
+    /// # Returns
+    ///
+    /// A string representing the Connection ID.
+    ///
+    /// # Errors
+    ///
+    /// Returns a connection manager error is the connection is unauthorized or
+    /// is the life cycle fails to add the connection.
```

Comment:
Two typos:  Change is to if:

"Returns a connection manager error is ..."
"... is the life cycle fails"